accounts/views.py

```diff
@@ -1069,3 +1065,33 @@ def pending(request):
     }
     tvars.update(paginate(request, pendings, settings.SOUNDS_PENDING_MODERATION_PER_PAGE))
     return render(request, 'accounts/pending.html', tvars)
+
+
+@csrf_exempt
+def donation_complete(request):
+    params = {'cmd': '_notify-validate'}
+    for key, value in request.POST.items():
+        params[key] =  value
```

Comment:
This seems a bit scary: you are passing absolutely all POST parameters to the paypal API. Wouldn't it make more sense to just pick the ones that are needed?